packages/block-library/src/image/index.php

```diff
@@ -268,6 +263,9 @@ class="lightbox-trigger"
  */
 function block_core_image_print_lightbox_overlay() {
 	$close_button_label = esc_attr__( 'Close' );
+	$dialog_label       = esc_attr__( 'Enlarged images' );
```

Comment:
This label now applies to both the single image and the group of images in the gallery. Previously, the message announced was way more accurate for a single image:

<img width="615" alt="Screenshot 2024-08-21 at 17 20 23" src="https://github.com/user-attachments/assets/9bd8840b-8b01-4528-ab6e-43c331310863">

Reply:
That's how it should work. [e4ebf7d](https://github.com/WordPress/gutenberg/pull/62906/commits/e4ebf7d629e313d696f6aeba718decebda132297) is a step in the right direction 👍🏻